py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -392,6 +392,17 @@ def aten_ops_sigmoid(
     )
 
 
+@dynamo_tensorrt_converter(torch.ops.aten.sym_size.int)
```

Comment:
Consider adding the following above:
https://github.com/pytorch/TensorRT/blob/6b8ccbc69f05de3b6275cac4613735934fb559b0/py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py#L62-L66